docker/scripts/dev_start.sh

```diff
@@ -246,7 +246,7 @@ function local_volumes() {
 
 ## customized docker cmd
 function do_docker_image_inspect() {
-    docker image inspect -f {{.Config.Image}} $1 &> /dev/null
+    docker image inspect -f "{{.Config.Image}}" $1 &> /dev/null
```

Comment:
How about `docker images --format "{{.Repository}}:{{.Tag}}" | grep "^$IMAGE$"`?
As the `inspect` is pretty heavy and might have bug: https://github.com/docker/for-linux/issues/397

Reply:
> docker/for-linux#397

Good. I just noticed the `-f` option should be quoted as hinted by shellcheck then.